packages/sandcastle/scripts/buildGallery.js

```diff
@@ -101,22 +120,49 @@ export function buildGalleryList(galleryDirectory, includeDevelopment = true) {
       continue;
     }
 
+    if (development && !labels.includes("Development")) {
+      labels.push("Development");
+    }
+
     output.entries.push({
       id: slug,
       title: title,
       thumbnail: thumbnail,
       description: description,
-      labels: labels ?? [],
+      labels: labels,
       isNew: false,
     });
     if (legacyId) {
       output.legacyIdMap[legacyId] = slug;
     }
+
+    const jsFile = readFileSync(`${galleryBase}/main.js`, "utf-8");
+    await index.addCustomRecord({
+      url: `?id=${slug}`,
+      content: jsFile,
```

Comment:
Can we surround this with `<code>` tags for better suggestion display?


```suggestion
      content: `<pre><code>${jsFile}</pre></code>`,
```